src/components/BSelect/BSelect.vue

```diff
@@ -61,25 +61,43 @@ export default {
   name: 'b-select',
   validator: null,
   props: {
+    /**
+    * Component Id
+    */
     id: {
       type: String,
       required: true
     },
+    /**
+    * Array of objects to be displayed as options.
```

Comment:
We should define here what attributes need to be set on those objects

Reply:
I'll add this in my refactor branch